red/log.js

```diff
@@ -73,11 +83,19 @@ var log = module.exports = {
 
     init: function(settings) {
         logHandlers = [];
-        var consoleSettings = {};
-        if (settings.logging) {
-            consoleSettings = settings.logging.console || {};
+        var loggerSettings = {};
+        var keys = Object.keys(settings.logging);
```

Comment:
This assumes `settings.logging` exists - if it doesn't exist, `Object.keys` throws an exception (which is what has broken the tests).